LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
```

Comment:
This commit has no DCO sign-off. It's must in every commit. You could squash this with the previous commit.